SD_DSC.adoc

```diff
@@ -2568,8 +2568,22 @@ The evaluator shall examine the KMD to ensure that:
 ==== Cryptographic Key Management (Extended - FCS_CKM_EXT)
 
 ===== FCS_CKM_EXT.3 Cryptograhpic Key Access
-[IMPORTANT]
-This is new
+
+====== TSS
+
+The evaluator shall verify that for each key selected cryptograhpic method the ST contains the corresponding SFR. The evaluator shall ensure the key access operations which utilize the specified cryptographic methods. Where multiple methods may be selected, the evaluator shall verify that each methods usage is described.
```

Comment:
"key selected cryptograhpic method"? Maybe you mean "selected cryptographic key access method"?

Reply:
"The evaluator shall ensure the key access operations which utilize the specified cryptographic methods." what actually needs to be ensured?

Reply:
"each methods usage" -> "each method's usage"